src/NovelAudioChannel.cpp

```diff
@@ -0,0 +1,82 @@
+//
+// Created by krjoh on 29/08/2019.
+//
+
+#include "NovelAudioChannel.h"
+#include <string>
+#include <map>
+
+namespace NovelRT {
+NovelAudioChannel::NovelAudioChannel(std::string input, bool isBgm, int channel){
+  if (isBgm)
+  {
+    _loadFile(input, 0);
+    _isMusic = true;
```

Comment:
This could be simplified to `_isMusic = isBgm`

Reply:
As a FYI, any NovelAudioChannel convos were resolved as this was meant to be removed. (I did read through to see what mistakes were there though 👍 )